lib/ansible/modules/file.py

```diff
@@ -521,12 +527,25 @@ def ensure_absent(path):
     b_path = to_bytes(path, errors='surrogate_or_strict')
     prev_state = get_state(b_path)
     result = {}
+    file_args = module.load_file_common_arguments(module.params)
+    changed = False
 
     if prev_state != 'absent':
         diff = initial_diff(path, 'absent', prev_state)
 
         if not module.check_mode:
             if prev_state == 'directory':
+                # Check if 'attributes' in file_args
+                current_attributes = file_args.get('attributes', '')
+                if '-a' in current_attributes:
```

Comment:
Looks like there's a case when `current_attributes is None`:
```python-traceback
Traceback (most recent call last):
  File "<stdin>", line 133, in <module>
  File "<stdin>", line 125, in _ansiballz_main
  File "<stdin>", line 73, in invoke_module
  File "/usr/local/lib/python3.9/runpy.py", line 225, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "/usr/local/lib/python3.9/runpy.py", line 97, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/usr/local/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/tmp/ansible_file_payload_unz_zxje/ansible_file_payload.zip/ansible/modules/file.py", line 1007, in <module>
  File "/tmp/ansible_file_payload_unz_zxje/ansible_file_payload.zip/ansible/modules/file.py", line 1001, in main
  File "/tmp/ansible_file_payload_unz_zxje/ansible_file_payload.zip/ansible/modules/file.py", line 540, in ensure_absent
TypeError: argument of type 'NoneType' is not iterable
```
*(https://dev.azure.com/ansible/ansible/_build/results?buildId=109805&view=logs&j=c9dc1718-7fc3-5e87-e6a6-571058301006&t=ff7b63cb-2767-53c3-7382-bd37bd9a7b6a&l=2143)*